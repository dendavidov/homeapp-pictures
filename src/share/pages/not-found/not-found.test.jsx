import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from './not-found';

describe('NotFound', () => {
  it('should render correctly', () => {
    const notFound = renderer.create(<NotFoundPage />).toJSON();
    expect(notFound).toMatchSnapshot();
  });
});
