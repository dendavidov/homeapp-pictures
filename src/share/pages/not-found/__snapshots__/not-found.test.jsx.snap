// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
<div>
  Error 404. Not Found
</div>
`;
